Library/Formula/libxslt.rb

```diff
@@ -17,8 +17,17 @@ class Libxslt < Formula
 
   depends_on 'libxml2'
 
+  head do
+    patch :DATA
+    url 'git://git.gnome.org/libxslt'
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
   def install
-    system "./configure", "--disable-dependency-tracking",
+    cmd = if build.head? then "./autogen.sh" else "./configure" end
```

Comment:
yes, I'd agree but unfortunately the `autogen.sh` is not included in the stable release archive and I wanted to keep it compatible. Moreover,  `autogen.sh` also calls `./configure` for the HEAD version.